pkg/apply/poller/poller.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package poller
+
+import (
+	"context"
+
+	"sigs.k8s.io/cli-utils/pkg/kstatus/polling"
+	pollevent "sigs.k8s.io/cli-utils/pkg/kstatus/polling/event"
+	"sigs.k8s.io/cli-utils/pkg/object"
+)
+
+// Poller defines the interface the applier needs to poll for status of resources.
```

Comment:
Can we put some information on the parameters here?

Reply:
Done.